crates/core/action-pipeline/src/estimator.rs

```diff
@@ -35,44 +39,62 @@ impl Estimator {
                 continue;
             };
 
-            if let ActionNode::RunTarget(_, target) = node {
-                let task_id = Target::parse(target).unwrap().task_id;
+            match node {
+                ActionNode::SetupTool(_)
+                | ActionNode::InstallDeps(_)
+                | ActionNode::InstallProjectDeps(_, _) => {
+                    install_duration += *task_duration;
```

Comment:
including install times now, otherwise, the comparison seems much faster. which is wrong since they also install.